src/main/java/com/odysseusinc/arachne/datanode/WebApplicationStarter.java

```diff
@@ -47,6 +48,7 @@
 
     public static void main(String... argc) {
 
+        SecurityContextHolder.setStrategyName(SecurityContextHolder.MODE_INHERITABLETHREADLOCAL);
```

Comment:
Isn't this a security gap since we utilize thread pools?
https://github.com/spring-projects/spring-security/issues/6856

Shouldn't we use e.g. `setTaskDecorator` instead?